src/functionalTest/java/uk/gov/hmcts/opal/steps/LocalJusticeArea.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.hmcts.opal.steps;
+
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import net.serenitybdd.rest.SerenityRest;
+
+import static net.serenitybdd.rest.SerenityRest.then;
+import static uk.gov.hmcts.opal.config.Constants.LJA_REF_DATA_URI;
+import static uk.gov.hmcts.opal.steps.BearerTokenStepDef.getToken;
+
+public class LocalJusticeArea extends BaseStepDef{
+
+    @When("I make a request to the LJA ref data api with")
+    public void iMakeARequestToTheOffenceRefDataApiWith() {
+        SerenityRest.given()
+            .accept("*/*")
+            .header("Authorization", "Bearer " + getToken())
+            .contentType("application/json")
+            .when()
+            .get(getTestUrl() + LJA_REF_DATA_URI).then().log().all();
+
+    }
+
+    @Then("the offence ref data response is {int}")
```

Comment:
Is this the best approach? you have step defs for offence and courts defined in the LJA step def.

Reply:
done the changes